### ISSUE TITLE
the settings of plots when moving between results are discrated

### ISSUE THREAD

**Issue by xhajnal** [sybila/biodivineGUI#66]
When I show imported results, then TCA results and after that go back to imported, the setting are gone

**Comment by martindemko** [comment]
Yes, it's normal behaviour. It would be very difficult to implement it to store settings.
